components/forms/Form/Form.tsx

```diff
@@ -1,37 +1,31 @@
 import React, { useEffect, useState } from "react";
-import { withFormik, FormikProps } from "formik";
+import { FormikProps, withFormik } from "formik";
 import { getFormInitialValues } from "@lib/formBuilder";
-import { validateOnSubmit, getErrorList, setFocusOnErrorMessage } from "@lib/validation";
+import { getErrorList, setFocusOnErrorMessage, validateOnSubmit } from "@lib/validation";
 import { submitToAPI } from "@lib/integration/helpers";
-import { useFormTimer } from "@lib/hooks";
-import { Button, Alert } from "../index";
+import { useExternalScript, useFlag, useFormTimer } from "@lib/hooks";
+import { Alert, Button } from "../index";
 import { logMessage } from "@lib/logger";
 import { useTranslation } from "next-i18next";
-import { FormValues, InnerFormProps, DynamicFormProps, Responses } from "@lib/types";
 import axios from "axios";
-import { useFlag, useExternalScript } from "@lib/hooks";
 import Loader from "../../globals/Loader";
 import classNames from "classnames";
+import { DynamicFormProps } from "@components/containers/DynamicForm/DynamicForm";
+import { Responses } from "@lib/types";
 
-declare global {
-  interface Window {
-    dataLayer: Array<unknown>;
-    grecaptcha: {
-      // Maybe a better way to do this
-      execute: (arg1: string | undefined, arg2: Record<string, unknown>) => Promise<string>;
-      ready: (arg: () => void) => void;
-    };
-  }
+export interface InnerFormProps {
+  children?: React.ReactNode;
+  language: string;
 }
 
 /**
  * This is the "inner" form component that isn't connected to Formik and just renders a simple form
  * @param props
  */
-const InnerForm: React.FC<InnerFormProps & FormikProps<FormValues> & DynamicFormProps> = (
-  props: InnerFormProps & FormikProps<FormValues> & DynamicFormProps
-) => {
-  const { children, handleSubmit, isSubmitting, formConfig } = props;
+const InnerForm: React.FC<
+  InnerFormProps & FormikProps<{ [key: string]: unknown }> & DynamicFormProps
+> = (props: InnerFormProps & FormikProps<{ [key: string]: unknown }> & DynamicFormProps) => {
+  const { children, handleSubmit, isSubmitting, formRecord } = props;
```

Comment:
My preference is to have a consistent form datastructure across the whole codebase. In this way you know that when working with a form record its the same everywhere rather than different across different components in the codebase. I'm open to discussion though

Reply:
I agree about the consistent data-structure across the codebase.  What I'm proposing is just a little fancier destructuring assignment that provides direct access to calling `formID`, `recapthcaID`, and `form` instead of always chaining then off of formRecord in the scope of this function.

Reply:
Thanks I understand what you mean now. Very good suggestion